crates/common/src/fuzz/mod.rs

```diff
@@ -0,0 +1,266 @@
+use crate::{
+    dotrain_add_order_lsp::LANG_SERVICES,
+    frontmatter::{try_parse_frontmatter, FrontmatterError},
+};
+use dotrain::{error::ComposeError, RainDocument, Rebind};
+use proptest::prelude::RngCore;
+use proptest::test_runner::{RngAlgorithm, TestRng};
+use rain_interpreter_bindings::IInterpreterStoreV1::FullyQualifiedNamespace;
+use rain_interpreter_eval::{
+    eval::ForkEvalArgs,
+    fork::{ForkCallError, ForkedEvm, NewForkedEvm},
+    trace::RainEvalResult,
+};
+use strict_yaml_rust::StrictYamlLoader;
+use thiserror::Error;
+
+#[derive(Clone, Debug)]
+pub struct RunnerScenario {
+    pub name: String,
+    pub entrypoint: String,
+    pub runs: u64,
+    pub rebinds: Vec<String>,
```

Comment:
can we not invent the word "rebinds" and just stick with "bind" which is used elsewhere?